### DISCUSSION TITLE
Casaos project died?

### DISCUSSION THREAD

**Discussion by markrieder** [IceWhaleTech/CasaOS#2042] [General]
This project has not been making progress lately. It looks like she will forget about him. Apart from simple installation, it does not offer any features suitable for homelab. Zimaos is better. Please listen to people. 
- RAID support 
- virtual machine 
- external access + own app

**Comment by Dev01-D** [comment]
CasaOS and ZimaOS are designed for different usecases. 

'HomeLab' is a VERY broad spectrum. For users that want an easy to setup manageable home server for docker management it is perfect.
This is what i would say would be entry to intermediate user. Being it is sitting on top of Debian you can make more advanced configurations utilising the underlaying OS which does allow more experienced users to 'tinker' with the system.

CasaOS is perfectly suitable for the intended market / design and those wanting alternative options such as webui raid control / VM can do so with ZimaOS.

CasaOS can work with RAID however this is done on the base OS rather than in the webui.

External access can be acheived using Cloudflare Tunnels (which i am currently using on CasaOS) and there is a custom app installation option within the webui (which again i am running for apps that arent in the app store)

CasaOS is still very much managed and maintained and will be for the forseeable future.
